src/executor/admin/SubmitJobExecutor.cpp

```diff
@@ -114,22 +85,46 @@ folly::Future<Status> SubmitJobExecutor::execute() {
                     return finish(std::move(v));
                 }
                 case meta::cpp2::AdminJobOp::SHOW_All: {
-                    nebula::DataSet v({"Job Id", "Command", "Status", "Start Time", "Stop Time"});
                     DCHECK(resp.value().__isset.job_desc);
                     if (!resp.value().__isset.job_desc) {
                         return Status::Error("Response unexpected");
                     }
                     const auto &jobsDesc = *resp.value().get_job_desc();
-                    for (const auto &jobDesc : jobsDesc) {
-                        v.emplace_back(nebula::Row({
-                            jobDesc.get_id(),
-                            meta::cpp2::_AdminCmd_VALUES_TO_NAMES.at(jobDesc.get_cmd()),
-                            meta::cpp2::_JobStatus_VALUES_TO_NAMES.at(jobDesc.get_status()),
-                            jobDesc.get_start_time(),
-                            jobDesc.get_stop_time(),
-                        }));
+                    auto spaceName = qctx()->rctx()->session()->space().name;
+                    if (cmd == meta::cpp2::AdminCmd::REBUILD_TAG_INDEX ||
+                        cmd == meta::cpp2::AdminCmd::REBUILD_EDGE_INDEX) {
+                        std::unordered_map<std::string, meta::cpp2::JobStatus> indexStatus;
+                        for (const auto &jobDesc : jobsDesc) {
+                            if (jobDesc.get_cmd() == cmd) {
+                                DCHECK_EQ(jobDesc.get_paras().size(), 2UL);
+                                if (jobDesc.get_paras()[1] == spaceName) {
+                                    indexStatus.emplace(jobDesc.get_paras()[0],
+                                                        jobDesc.get_status());
+                                }
+                            }
+                        }
+                        nebula::DataSet v({"Name", "Index Status"});
```

Comment:
Why `show tag index status` is implemented in `submit job`? BTW, the storage PR also confuses me, please explain a bit more.

Reply:
Because `rebuild index` is moved to jobManager, so If I want to show tag/index status after rebuilding index, I need to filter the results that `show jobs` return to get the index status.

Reply:
I know what you said, if so, for example, why not write `start` when rebuild index start, and write `done` when finished?